README.md

```diff
@@ -1,4 +1,11 @@
-# Prometheus
+<h1 align="center" style="border-bottom: none">
+    <a href="//prometheus.io" target="_blank"><img alt="Prometheus" src="/documentation/images/Prometheus_software_logo.svg"></a><br>Prometheus
```

Comment:
Let's just call it `prometheus-logo.svg` in this repo, the name from Wikipedia is a bit clunky :)

Reply:
haha got it

Reply:
> Let's just call it `prometheus-logo.svg` in this repo, the name from Wikipedia is a bit clunky :)

done, hope it works!